source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
+    * Reconstructed event X-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``DETY`` tform: ``1E``, unit: deg
+    * Reconstructed event Y-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``EVENT_TYPE`` tform: ``32X``
+    * Event quality partition.
+
+      .. note::
+         It needs to be discussed whether this should be a mandatory column
+         or an optional column. The partition of events into quality classes
+         seems sufficiently fundamental so that all IACT should in principle
+         support this. For example event multiplicity could be translated
+         into quality partitions, and the IRFs computed accordingly.
+
+
+Optional columns
+----------------
+
+.. note::
+   None of the following columns is required to be part of an ``EVENTS``
+   extension. Any software using these columns should first check whether the
+   columns exist, and warn in case of their absence.
+
+* ``ALT`` tform: ``1E``, unit: deg
+    * Reconstructed altitude coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``AZ`` tform: ``1E``, unit: deg
+    * Reconstructed azimuth coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``THETA`` tform: ``1E``, unit: deg
+    * Reconstructed offset from the observation pointing position
+* ``PHI`` tform: ``1E``, unit: deg
+    * Reconstructed position angle from the observation pointing position
+      (position angles are counted counterclockwise from celestial North)
+* ``MULTIP`` tform: ``1I``
+    * Telescope multiplicity. Number of telescopes that have seen the event
+* ``DIR_ERR`` tform: ``1E``, unit: deg
+    * Direction error of reconstruction 
+* ``ENERGY_ERR`` tform: ``1E``, unit: TeV
+    * Error on reconstructed event energy
+* ``COREX`` tform: ``1E``, unit: m
+    * Core position X of shower
+* ``COREY`` tform: ``1E``, unit: m
+    * Core position Y of shower
+* ``CORE_ERR`` tform: ``1E``, unit: m
+    * Error on core position of shower    
+* ``XMAX`` tform: ``1E``, unit: radiation lengths
+    * First interaction depth 
+* ``XMAX_ERR`` tform: ``1E``, unit: radiation lengths
+    * Error on first interaction depth 
+* ``HIL_MSW`` tform: ``1E``
+    * Hillas mean scaled width
+* ``HIL_MSW_ERR`` tform: ``1E``
+    * Hillas mean scaled width error
+* ``HIL_MSL`` tform: ``1E``
+    * Hillas mean scaled length
+* ``HIL_MSL_ERR`` tform: ``1E``
+    * Hillas mean scaled length error
+
+
+Mandatory header keywords
+-------------------------
```

Comment:
> I agree that DEADC could also be moved to into the pointing table (it should not go into GTI as these may be changed by the user). You don't need to do that for LIVETIME, you even do not need LIVETIME as it can be inferred from DEADC and ONTIME.

Ah, this wasn't clear to me.

If you remove RA_PNT and DEC_PNT from the EVENTS header, then POINTING becomes required anyways, because science tools need to know the pointing position to use background models.
I'm OK with this break, but then let's move LIVETIME info to POINTING as well now.
@jknodlseder - Is it OK for you ... e.g. ctobssim will always have to write POINTING tables.

This is getting closer to the Fermi-LAT data model.
Maybe include a reference to this and say that our POINTING is similar to the Fermi-LAT SPACECRAFT?
http://fermi.gsfc.nasa.gov/ssc/data/analysis/documentation/Cicerone/Cicerone_Data/LAT_Data_Columns.html#SpacecraftFile

Also, I think if it's going this way, changing in POINTING from TIME to (START, STOP) is better, because LIVETIME or DEADC is valid for a time interval. Overall moving to START, STOP and saying that the quantities are associated with START isn't a big change.

@jknodlseder - Do you agree that would be a good change to make to POINTING? Assuming yes -- Do you want to do it in this PR or should I do it in a follow-up PR tomorrow?

How does Fermi-LAT handle deadtime information? Is it in the spacecraft file?

To be able to